incubator/virtualcluster/pkg/syncer/controllers/pod/controller.go

```diff
@@ -52,7 +56,9 @@ func Register(
 	controllerManager *manager.ControllerManager,
 ) {
 	c := &controller{
-		client: client,
+		client:  client,
+		queue:   workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "super_master_pod"),
+		workers: 1,
```

Comment:
If we use workers, let us make default to a value > 1 . 
check http://openkruise.io/en-us/blog/blog2.html